finalized DELETE route for warehouses - tested

victorarabi opened the pull request:
Correct PR request to develop instead of main

ghost commented:
Victor if you can pull the new developed branch and then merge it into ur branch and push code again but dont close or create new PR that will be added to this one automatically

victorarabi commented:
hey @kaushal0110 done!

kaushal0110 commented on routes/deleteWarehouse/deleteWarehouse.js:
```diff
@@ -0,0 +1,43 @@
+});
+
+//DELETE request
+deleteWarehouse.delete('/:location', (req, res, next) => {
```
So instead of getting the warehouse name and then changing it to lowercase can we use the id of warehouse for searching the warehouse from file

kaushal0110 commented on routes/deleteWarehouse/deleteWarehouse.js:
```diff
@@ -0,0 +1,43 @@
+deleteWarehouse.delete('/:location', (req, res, next) => {
+  let warehouseToDelete = req.params.location;
+  //checks if requested warehouse exists on database
+  let warehouseCheck = warehouses.filter(
```
So instead if alll the code from here to line 33 we could have simplified it in an easier way i know u r checking first if warehouse exists or not but if we use the find function we can just say if find function not return something then dont proceed else do some thing

victorarabi commented:
hi @kaushal0110 I have added the callback to check the read/write and changed from filesync to just file. Your other suggestions are not impeditive for the operation of this branch. I won't have time now to modify this and I will ping it as an improvement if we have time.
Please remember that Agile means that we have it working and it is.

victorarabi commented:
@kaushal0110 this should be good to go

ghost commented:
> hi @kaushal0110 I have added the callback to check the read/write and changed from filesync to just file. Your other suggestions are not impeditive for the operation of this branch. I won't have time now to modify this and I will ping it as an improvement if we have time.
> Please remember that Agile means that we have it working and it is.

okay but now your branch has merge conflicts because Kevin merged his branches earlier if you could resolve that and push again thnx

victorarabi commented:
There should be no conflicts anymore @kaushal0110

kaushal0110 commented on data/warehouses.json:
```diff
@@ -76,4 +89,21 @@
+      "email": "vmendoza@instock.com"
+    }
   }
+<<<<<<< HEAD
```
I think you left this just remove this I will approve it
